_posts/2014-10-07-socratic-seminar-30.md

```diff
@@ -53,7 +53,7 @@ Weekly Mining Stats
 [](http://organofcorti.blogspot.se/2014/10/october-6th-2014-weekly-bitcoin-network.html)[http://organofcorti.blogspot.se/2014/10/october-6th-2014-weekly-bitcoin-network.html](http://organofcorti.blogspot.se/2014/10/october-6th-2014-weekly-bitcoin-network.html)
 
 Building a block-chain database the right way  
-[](http://blog.coinspect.co/building-block-chain-database-right-way)[http://blog.coinspect.co/building-block-chain-database-right-way](http://blog.coinspect.co/building-block-chain-database-right-way)
+[](https://www.coinspect.com/blog/building-block-chain-database-right-way/)[[https://www.coinspect.com/blog/building-block-chain-database-right-way/](https://www.coinspect.com/blog/building-block-chain-database-right-way/)](https://www.coinspect.com/blog/building-block-chain-database-right-way/)
```

Comment:
I’m not sure why the links are all in here with an empty link text and repeated in full, but this introduces a new unnecessary opening bracket that should be dropped.

```suggestion
[](https://www.coinspect.com/blog/building-block-chain-database-right-way/)[https://www.coinspect.com/blog/building-block-chain-database-right-way/](https://www.coinspect.com/blog/building-block-chain-database-right-way/)](https://www.coinspect.com/blog/building-block-chain-database-right-way/)
```